src/index.html

```diff
@@ -188,5 +188,47 @@ <h1>Made with love.</h1>
       ga('create', 'UA-67015100-1', 'auto');
       ga('send', 'pageview');
     </script>
+    <a href="https://your-url" class="github-corner" aria-label="View source on GitHub"><svg
```

Comment:
This looks like it is effortlessly copy-and-pasted. I would recommend having a look at what you pasted and fill in the blanks.

Reply:
Oops! Sorry, I completely forgot about this :man_facepalming:

Reply:
Thanks ok @lex111.